patches/minecraft/net/minecraft/client/renderer/entity/layers/LayerArmorBase.java.patch

```diff
@@ -40,9 +40,13 @@
 +                    } // Default
 +                        if (!this.field_177193_i && itemstack.func_77962_s())
                          {
-                             func_188364_a(this.field_177190_a, p_188361_1_, t, p_188361_2_, p_188361_3_, p_188361_4_, p_188361_5_, p_188361_6_, p_188361_7_, p_188361_8_);
+-                            func_188364_a(this.field_177190_a, p_188361_1_, t, p_188361_2_, p_188361_3_, p_188361_4_, p_188361_5_, p_188361_6_, p_188361_7_, p_188361_8_);
++                            if(net.minecraftforge.client.ArmorOverlayHandler.run(field_177190_a.func_177068_d().field_78724_e, itemstack, p_188361_1_, p_188361_4_, p_188361_9_, t, p_188361_2_, p_188361_3_, p_188361_6_, p_188361_7_, p_188361_8_))
++                                func_188364_a(this.field_177190_a, p_188361_1_, t, p_188361_2_, p_188361_3_, p_188361_4_, p_188361_5_, p_188361_6_, p_188361_7_, p_188361_8_);
```

Comment:
Don't indent (smaller patch). And comment what this edit is doing. Don't want to go dig up this PR if I'm confused about why this is here.